core/domain/user_jobs_continuous.py

```diff
@@ -608,37 +598,37 @@ def map(item):
                 # Find score for this specific exploration.
                 exploration_data.update({
                     'exploration_impact_score': (
-                        value_per_user * reach * contribution)
+                        value_per_user * reach * contribution),
```

Comment:
Here and elsewhere (some places above) -- I'm not actually that keen on all these trailing commas, especially for dicts that we shouldn't be updating in the future. (The exception would be dicts at the controller boundary.) Can we not add them?